client/src/components/common/ProfileCard/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Button from '../Button';
+import './style.css';
```

Comment:
add a new line before 
```suggestion

import './style.css';
```